lua/mellifluous/highlights/plugins/nvim_notify.lua

```diff
@@ -0,0 +1,23 @@
+local M = {}
+
+function M.set(hl, colors)
```

Comment:
I would recommend these highlights:
```lua
    hl.set("NotifyERRORBorder", { fg = colors.fg5 })
    hl.set("NotifyWARNBorder", { fg = colors.fg5 })
    hl.set("NotifyINFOBorder", { fg = colors.fg5 })
    hl.set("NotifyDEBUGBorder", { fg = colors.fg5 })
    hl.set("NotifyTRACEBorder", { fg = colors.fg5 })

    hl.set("NotifyERRORIcon", { fg = colors.ui_red })
    hl.set("NotifyWARNIcon", { fg = colors.ui_orange })
    hl.set("NotifyINFOIcon", { fg = colors.ui_blue })
    hl.set("NotifyDEBUGIcon", { fg = colors.ui_purple })
    hl.set("NotifyTRACEIcon", { fg = colors.ui_orange })

    hl.set("NotifyERRORTitle", { fg = colors.fg2 })
    hl.set("NotifyWARNTitle", { fg = colors.fg2 })
    hl.set("NotifyINFOTitle", { fg = colors.fg2 })
    hl.set("NotifyDEBUGTitle", { fg = colors.fg2 })
    hl.set("NotifyTRACETitle", { fg = colors.fg2 })
```
It has a more minimal look with less colors and darker borders. By using `ui_*` colors, it can be ensured, that every color will have the same brightness (otherwise color sets with inconsistent brightness like alduin could produce strange results). Also without bg color the highlights automatically support transparency.

Light bg should normally automatically work when using any of the predefined colors.

Reply:
Thank you for the guidance. I will update shortly.

Reply:
Can you also remove the bg color from highlights? so they can be transparent